notebooks/readme.md

```diff
@@ -0,0 +1,15 @@
+Most of the notebooks in this folder may be opened with Google Colaboratory. They represent
+experiments to demonstrate the base classes and basic functionality of libraries contained in 
+rest of the repoository. Collectively they are intended to support incorporation of evaluation 
+of the dynamic routing codes and related utilities being considered and prepared as potential
+candidates for use as the base dynamic hydraulic engine for the NWM. 
+
+In general, as with all things related to the National Water Center, our goal is to save
```

Comment:
Is this needed in the notebooks readme separately?

Reply:
Good questions (Nels had some too.) To respond, do I make new changes to my fork, recommit/push/PR? (That's easy enough, but I'm not sure if that preserves the trail of these comments here, for instance.)

Reply:
It's up to you, whatever is easiest and usually that's the most direct path since these aren't hard tasks, just housekeeping.

Reply:
Just keep pushing commits to the branch you are asking to pull, and they keep showing up here, once we get things settled for an initial point, we will merge this PR and pick up outstanding issues/development in new PRs.

Reply:
Moved the contents of the referenced notebooks readme. Working on a few others.